tratto/src/main/java/star/tratto/util/JavaParserUtils.java

```diff
@@ -283,10 +283,10 @@ public static Set<MethodUsage> getMethodsOfType(String type) throws IllegalArgum
         } catch (UnsupportedOperationException e) {
             if (!resolvedType.isArray()) {
                 throw new IllegalArgumentException("Trying to retrieve available methods from a type that is not " +
-                        "a reference type or an array: " + type, e);
+                        "a reference type or an array: " + referenceType, e);
             }
         } catch (UnsolvedSymbolException e) {
-            logger.warn("Unresolvable type: {}", type);
+            logger.warn("Unresolvable type: {}", referenceType);
         } finally {
             if (useObjectMethods) {
```

Comment:
Why is this code in a `finally` block rather than just being after the `try` block?  The code doesn't have any effect (except using CPU) if an exception is being thrown.